onnxruntime/test/global_thread_pools/test_fixture.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/session/onnxruntime_cxx_api.h"
+#include <gtest/gtest.h>
+
+#ifdef _WIN32
+typedef const wchar_t* PATH_TYPE;
+#define TSTR(X) L##X
+#else
+#define TSTR(X) (X)
+typedef const char* PATH_TYPE;
+#endif
+
+//empty
```

Comment:
Can we use core/common/path_string.h?

Reply:
This was copied from test_fixture.h in shared_lib. Actually I don't even need this file since I'm including the one from shared_lib. will delete this.